.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+on: [push, pull_request]
+
+env:
+  TZ: "Europe/Amsterdam"
+  CI: "true"
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        job: [lint, test]
```

Comment:
For completeness sake, I would add the `build` step here as well, in case changes have been made to Webpack's configuration files or other changes that might impact building the project's assets.

Reply:
I added this here, but be aware the current Jenkins deployment also only runs linting and testing.

Reply:
You're right, but if we can optimise the time that it takes to run the actions, we can use Jenkins solely for deployment which should speed up the build/deploy as a whole.